src/main/java/de/rwth/idsg/steve/web/dto/WebUserForm.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * SteVe - SteckdosenVerwaltung - https://github.com/steve-community/steve
+ * Copyright (C) 2013-2024 SteVe Community Team
+ * All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package de.rwth.idsg.steve.web.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+
+ /**
+ * @author fnkbsi
+ * @since 01.04.2022
+ */
+@Getter
+@Setter
+public class WebUserForm {
+
+    // Internal database id
+    //private Integer webUserPk;
+
+    private Boolean enabled;
+
+    private String webusername;
+
+    private String password;
+
+    private String passwordComparison;
+
+    private String authorities;
+
+    private Boolean pwerror;
+
+    private String apitoken;
```

Comment:
- pls use camel case in fields
- pwerror should not be needed after refactoring to use validation annotations.